### PR TITLE
chore(docs): release notes 2020.15

### LINKED ISSUE #5154: release notes 2020.15
**Describe the issue. What is the expected and unexpected behavior?**
release notes 2020.15

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5154 

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,92 @@ id: Release notes
...
+  - Moved page html demos to navigation component ([#5109](https://github.com/patternfly/patternfly-react/pull/5109))
+- **Skeleton:** 
+  - Updated skeleton examples ([#5072](https://github.com/patternfly/patternfly-react/pull/5072))
+- **Splitter:** 
+  - Removed splitter component ([#5105](https://github.com/patternfly/patternfly-react/pull/5105))
+- **Tree View:**
+  -  Added support for indeterminate checkboxes without requiring consumer to use refs ([#5150](https://github.com/patternfly/patternfly-react/pull/5150))
+- **Toolbar:** 
+  - Added flag to match page insets ([#5098](https://github.com/patternfly/patternfly-react/pull/5098))
+  - updated breakpoints and formatting examples. Created collapsed variant. ([#5123](https://github.com/patternfly/patternfly-react/pull/5123))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:52]: Change to "Updated"

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,92 @@ id: Release notes
...
+
+### Virtualized extension
+- Added virtualized examples with composable components ([#5122](https://github.com/patternfly/patternfly-react/pull/5122))
+
+### Other
+-  **Chore:**
+  - Cleanup dependencies ([#5060](https://github.com/patternfly/patternfly-react/pull/5060))
+  - Bumped Victory package ([#5116](https://github.com/patternfly/patternfly-react/pull/5116))
+  - Fixed circular deps ([#5086](https://github.com/patternfly/patternfly-react/pull/5086))
+  - Fixed example codesandbox ([#5065](https://github.com/patternfly/patternfly-react/pull/5065))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:83]: The tabbing for these items is off.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,92 @@ id: Release notes
...
+- **Action list:** 
+  - Added action list component ([#5112](https://github.com/patternfly/patternfly-react/pull/5112))
+- **Card:** 
+  - Added expandable card, move tests ([#5093](https://github.com/patternfly/patternfly-react/pull/5093))
+  - Added keyboard interaction a11y test, clean up docs ([#5082](https://github.com/patternfly/patternfly-react/pull/5082))
+- **Checkbox:**
+  - Fixed Typescript type for isChecked prop so it allows null (indeterminate) ([#5148](https://github.com/patternfly/patternfly-react/pull/5148))
+- **Data list:** 
+  - Fixed to respect isDisabled for dragging ([#5052](https://github.com/patternfly/patternfly-react/pull/5052))
+- **DualListSelector:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:28]: Sentence case

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,92 @@ id: Release notes
...
+  - Added/updated cypress tests for modal ([#5081](https://github.com/patternfly/patternfly-react/pull/5081))
+- **Navigation:** 
+  - Moved page html demos to navigation component ([#5109](https://github.com/patternfly/patternfly-react/pull/5109))
+- **Page:** 
+  - Moved page html demos to navigation component ([#5109](https://github.com/patternfly/patternfly-react/pull/5109))
+- **Skeleton:** 
+  - Updated skeleton examples ([#5072](https://github.com/patternfly/patternfly-react/pull/5072))
+- **Splitter:** 
+  - Removed splitter component ([#5105](https://github.com/patternfly/patternfly-react/pull/5105))
+- **Tree View:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:48]: Sentence case

**Review by evwilkin — CHANGES_REQUESTED** [review]
sentence case needed for Dual list selector & Tree view, and the indentation is off for the items under "Chore" at the bottom as @redallen noted.

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -3,6 +3,92 @@ id: Release notes
...
+  -  Added support for indeterminate checkboxes without requiring consumer to use refs ([#5150](https://github.com/patternfly/patternfly-react/pull/5150))
+- **Toolbar:** 
+  - Added flag to match page insets ([#5098](https://github.com/patternfly/patternfly-react/pull/5098))
+  - pdated breakpoints and formatting examples. Created collapsed variant. ([#5123](https://github.com/patternfly/patternfly-react/pull/5123))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:52]: Missing the "U" in "Updated".

**Review by redallen — CHANGES_REQUESTED** [review]

**Comment by evwilkin** [comment]
PR #5158 has now been merged for this release as well.

**Review by redallen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Pull request merged**